SayTheirNames/Source/STNAppDelegate.swift

```diff
@@ -37,6 +39,10 @@ class STNAppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         NSSetUncaughtExceptionHandler(exceptionHandler)
         
+        // Setup SDWebImage Plugins
+        SDImageLoadersManager.shared.addLoader(SDImageLinkLoader.shared)
+        SDWebImageManager.defaultImageLoader = SDImageLoadersManager.shared
```

Comment:
How does it affect regular image loading if this is set as a `defaultImageLoader`?

Reply:
@hybridcattt from my testing it didn't seem to affect normal image loading - I don't know a ton about the internals of SDWebImage but images were cached/loaded appropriately during testing. I'm not sure if it's possible with SDWebImage, but I'd feel a lot better just using it for the URLs if possible.

Reply:
completely agree! I think it's worth looking into.